sections/slideshow.liquid

```diff
@@ -1,4 +1,5 @@
 {{ 'section-image-banner.css' | asset_url | stylesheet_tag }}
+{{ 'component-rte.css' | asset_url | stylesheet_tag }}
```

Comment:
The footer is loading the `component-rte.css` asset. I think we should just remove it from anywhere else since the footer is always present anyway 🤷 
Otherwise, not a huge deal but this in the inspect tool is pretty annoying: 
<details>
<summary>Screenshot</summary>

![](https://screenshot.click/22-18-1e7zr-ncvz7.png)
</details>